templates/forms/_talk.twig

```diff
@@ -56,6 +57,7 @@
                 <option value="entry" {% if level == 'entry' %}selected {% endif %}>Entry level</option>
                 <option value="mid" {% if level == 'mid' %}selected {% endif %}>Mid-level</option>
                 <option value="advanced" {% if level == 'advanced' %}selected {% endif %}>Advanced</option>
+                <option value="mixed" {% if level == 'mixed' %}selected {% endif %}>Soft Skills/All levels</option>
```

Comment:
I don't think a separate category is required here, I think it's implied that if it's a soft talk, the category doesn't matter anyway